hvac/v1/__init__.py

```diff
@@ -978,6 +977,80 @@ def auth_approle(self, role_id, secret_id=None, mount_point='approle', use_token
 
         return self.auth('/v1/auth/{0}/login'.format(mount_point), json=params, use_token=use_token)
 
+    def create_kubernetes_configuration(self, host="", cert=None, jwt_token=None, pem_keys=[], mountpoint='kubernetes'):
+        """
+        POST /auth/kubernetes/config
+        """
+
+        url = 'v1/auth/{0}/config'.format(mount_point)
+        params = {
+            'host': host,
+            'kubernetes_ca_cert': cert,
+        }
+        if jwt_token is not None: params['token_reviewer_jwt'] = jwt_token
+        if len(pem_keys) > 0: params['pem_keys'] = pem_keys
+        return self._post(url, json=params).json()
+
+    def get_kubernetes_configuration(self, mount_point='kubernetes'):
+        """
+        GET /auth/kubernetes/login
+        """
+
+        url = '/v1/auth/{0}/config'.format(mount_point)
+        return self._get(url).json()
+
+    def auth_kubernetes(self, host, role, jwt, mount_point='kubernetes'):
```

Comment:
Thanks for adding Kubernetes auth!

`host` parameter in `auth_kubernetes` is not used. What is it intended for?